### PR TITLE
Initial commentary

### PR DESCRIPTION
**Pull request by dfudenb**
discuss!

re LaTeX packages checkout
https://ctan.org/pkg/cooking-units
and
https://ctan.org/pkg/makecookbook

the cookbooky and cuisine seem a little rigid / not quite what is desired.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,16 @@
-# git-cookin project readme/notes
+# git-cookin
+
+A repository dedicated to preserving the best flavors of 
+[potluck club]("known aliases GOT potluck, Naptime potluck"),
+for likely eventual transoformation into some physical entity.
```
Comment (kunalsahasrabuddhe) [inline README.md]: Typo (transformation)

Also, unclear what this line means - presumably physical entity = cookbook, but could mean a dish, or ... other?

File: README.md [hunk 2]
```diff
@@ -1 +1,16 @@
...
+
+## Ingredients
+
+* Every recipe herein must have been cooked for some potluck by a `primary chef`
+* Every recipe must have an awesome name
```
Comment (kunalsahasrabuddhe) [inline README.md]: Whoa this'd be cool and really stretch our imaginations!

File: recipes/readme.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# recipes
+
+This directory is organized by primary author of a recipe.
```
Comment (kunalsahasrabuddhe) [inline recipes/readme.md]: Unless this gets unwieldy, easier to have a single directory for all recipe tex files. In the makefile we'd put in a tex file source directory and not have to deal with subdirectories.

Thoughts?

File: templates/readme.md [hunk 4]
```diff
@@ -0,0 +1 @@
+# syle here?
```
Comment (kunalsahasrabuddhe) [inline templates/readme.md]: I think it's a good idea to have separate .sty files

Let me look at the dancebook project structure I mentioned to see how that's structured. it works well with the makefile.

Will also try to get permission to use that makefile here with modifications (though it's cc licensed, so should be no problem)

File: README.md (hunk 2 above)
Comment (dfudenb) [reply README.md]: or measurably awesome name --

File: templates/readme.md (hunk 4 above)
Comment (dfudenb) [reply templates/readme.md]: 👍

File: README.md (hunk 1 above)
Comment (dfudenb) [reply README.md]: true story -- i suppose books have bindings -- 
etc

so physical cookbook then

File: recipes/readme.md (hunk 3 above)
Comment (kunalsahasrabuddhe) [reply recipes/readme.md]: On second thought, separating by author allows people to compile their own recipes (see comment on other commit) more easily by using makefiles that only point to their subdirs.

File: collations/readme.md [hunk 5]
```diff
@@ -0,0 +1 @@
+# do we support different selections of recipes into a pdf?
```
Comment (kunalsahasrabuddhe) [inline collations/readme.md]: I think so, but this'll have to have individual switches for various recipes in the makefile.

The easy selection to support is by author. Anything else will have to look at some kind of tag or label within a file to decide whether to select it. Can be done but I don't have make-fu for this (yet).

File: README.md [hunk 6]
```diff
@@ -14,3 +14,17 @@ to be included in the final binding, as such we will discuss some (probably mult
...
+.* title
+.* author
+.* tags
+.* ingredients
+.* steps
+.* estimated servings
+.* estimated time (total, elapsed, active)
+.* commentary and [flavor text]("not neccessarily about the actual flavor")
+.* required equipment (desired, necessary)
+.* date first brought to potluck (or `BGC` if before git-cookin) and the theme of that day if applicable
```
Comment (kunalsahasrabuddhe) [inline README.md]: Is date necessary? Not totally opposed, but this imposes overhead.

File: README.md [hunk 7]
```diff
@@ -14,3 +14,17 @@ to be included in the final binding, as such we will discuss some (probably mult
...
+1. move info into tex file
```
Comment (kunalsahasrabuddhe) [inline README.md]: I'm unclear on what these are steps for - adding a recipe to the repo? I'd hope people would dump text directly into tex file, then format! Only real requirement is making sure it compiles with all necessary fields filled out before review for merge.

File: recipes/readme.md (hunk 3 above)
Comment (dfudenb) [reply recipes/readme.md]: organizing by author directory can grant additional ownership over those files -- eg codeowner

and we can still import all files by name directly if we just put all the subdirectories on the file path -- 
\makeatletter
\def\input@path{{/path/folder1/}{/path/folder2/}}
\makeatother

-- which would only require code change when new chefs arrive. 

-- though we would still have to enforce separate recipe names at the file level 

organizing into a chef per dir can also allow for creations of bindings of just so-and-so's recipes 

i'm leaning towards the benefits -- but could be talked down

File: README.md [hunk 8]
```diff
@@ -12,19 +16,54 @@ for likely eventual transoformation into some physical entity.
...
 to be included in the final binding, as such we will discuss some (probably multiple) filters
 
-## Steps 
+## Steps
 
 1. dump recipe text
 1. identify important sections
 .* title
 .* author
+..* this shouldn't need an example...
```
Comment (kunalsahasrabuddhe) [inline README.md]: Really shouldn't. Suggest removing this line, unless we want to enforce author format (in which case put in an actual example).

File: README.md (hunk 7 above)
Comment (dfudenb) [reply README.md]: yep -- for going about adding a recipe -- 
and to help organize our thoughts on what are important sections and flavor for recipes in this repo
i'll squash last line into first line

File: README.md [hunk 9]
```diff
@@ -12,19 +16,54 @@ for likely eventual transoformation into some physical entity.
...
 .* tags
```
Comment (kunalsahasrabuddhe) [inline README.md]: These are key - what's the best way to include tags in a tex file so it's searchable by makefile? Worth a look!

File: collations/readme.md (hunk 5 above)
Comment (dfudenb) [reply collations/readme.md]: or several different master make files

File: README.md [hunk 10]
```diff
@@ -12,19 +16,54 @@ for likely eventual transoformation into some physical entity.
...
 .* required equipment (desired, necessary)
 .* date first brought to potluck (or `BGC` if before git-cookin) and the theme of that day if applicable
+.* source of inspiration / credit?
 1. move info into tex file 
 
+--- 
+
+## discussion of further requirements
+* must be able to compile individual recipes with minimal hassle
+* must have full list of dependencies to build recipes on their own
```
Comment (kunalsahasrabuddhe) [inline README.md]: List out dependencies
- Images in an images folder
- All style files up to date and all syntax matching style files
- Other?

File: collations/readme.md (hunk 5 above)
Comment (dfudenb) [reply collations/readme.md]: and not necessary to build to pdf all files 

-- assuming that pdfs are the standard output format -- 

then maybe make an editions folder too for blessed output of the best pdf?

File: README.md [hunk 11]
```diff
@@ -12,19 +16,54 @@ for likely eventual transoformation into some physical entity.
...
+* must figure out how to register approval -- what sort of criteria are important to get 
+.* do we need some other database / table that records what people think of each recipe
```
Comment (kunalsahasrabuddhe) [inline README.md]: What's the best way to implement this while keeping it easy for people to add to it? Want to minimise overhead beyond knowledge of LaTeX, ideally. However, such a record is a decent idea for historical recreation.

File: README.md [hunk 12]
```diff
@@ -12,19 +16,54 @@ for likely eventual transoformation into some physical entity.
...
+.* or just a chef + approver system to get things in to the mater cookbook
```
Comment (kunalsahasrabuddhe) [inline README.md]: Probably sufficient.

Also, please fix typo.

File: README.md (hunk 6 above)
Comment (dfudenb) [reply README.md]: recommended and it helps us get into the theme of -- 
this dish must be cooked at a potluck to qualify for entry 
-- and is an interesting thing to collect
as then best dishes from meals could be reconstructed

File: README.md (hunk 9 above)
Comment (dfudenb) [reply README.md]: 👀

File: README.md [hunk 13]
```diff
@@ -12,19 +16,54 @@ for likely eventual transoformation into some physical entity.
...
 
+--- 
+
+## discussion of further requirements
+* must be able to compile individual recipes with minimal hassle
+* must have full list of dependencies to build recipes on their own
+* must figure out how to register approval -- what sort of criteria are important to get 
+.* do we need some other database / table that records what people think of each recipe
+.* or just a chef + approver system to get things in to the mater cookbook 
+* may want to standardize on a set of units that are approved and those that are verboten as well as standard aliases in the latex
```
Comment (kunalsahasrabuddhe) [inline README.md]: If we use cooking-units, it might be possible to enforce this per-compile to save people the hassle of converting everything. I'll look into it.

File: README.md (hunk 10 above)
Comment (dfudenb) [reply README.md]: or do images live in a persons recipe folder?

how should one go about checking the style file up to date-ness

File: README.md [hunk 14]
```diff
@@ -12,19 +16,54 @@ for likely eventual transoformation into some physical entity.
...
+.+ cups
+.+ liters
+.+ Tablespoons
+.+ Teaspoons
+.- pinches
+.- to taste
+.- pecs
```
Comment (kunalsahasrabuddhe) [inline README.md]: What?

File: README.md [hunk 15]
```diff
@@ -12,19 +16,54 @@ for likely eventual transoformation into some physical entity.
...
+.+ lbs
+.+ grams
+.* 
+* how to reference other recipes from recipes
```
Comment (kunalsahasrabuddhe) [inline README.md]: Maybe let's keep this in here but punt on implementing?

File: README.md [hunk 16]
```diff
@@ -12,19 +16,54 @@ for likely eventual transoformation into some physical entity.
...
+* support for multiple page recipes,
```
Comment (kunalsahasrabuddhe) [inline README.md]: YES!

This should be a requirement for the stylefile and the makefile

File: README.md [hunk 17]
```diff
@@ -12,19 +16,54 @@ for likely eventual transoformation into some physical entity.
...
+.- to taste
+.- pecs
+.+ lbs
+.+ grams
+.* 
+* how to reference other recipes from recipes
+* support for multiple page recipes, 
+* # of columns to use in the style
+.+ 1
+.- 2
```
Comment (kunalsahasrabuddhe) [inline README.md]: I think 2 cols ok for, say, ingredient list, or some place adding info in a second column adds value.
For recipe body, I agree we stick to single.

File: README.md (hunk 11 above)
Comment (dfudenb) [reply README.md]: honestly we don't have a ridiculous amount of recipes / items -->
however impressions of what a dish are do vary on when it was remade --
ie if sometimes the dish comes out better

however we could get pretty far with just like a csv file / .xlsx -- but... 
not sure the best way to make easy to add to and organize with the recipes.

File: README.md (hunk 12 above)
Comment (dfudenb) [reply README.md]: ah i meant miter -- a cookbook for people with big hats

File: README.md [hunk 18]
```diff
@@ -12,19 +16,54 @@ for likely eventual transoformation into some physical entity.
...
+* how to organize the compilations
```
Comment (kunalsahasrabuddhe) [inline README.md]: This again is a question of how much we can customise makefiles and have them auto-populate compilations. Anything more will require making new makefiles or top level tex files, which strikes me as a pain and therefore something people won't do much.

File: README.md [hunk 19]
```diff
@@ -12,19 +16,54 @@ for likely eventual transoformation into some physical entity.
...
+.* 
+* how to reference other recipes from recipes
+* support for multiple page recipes, 
+* # of columns to use in the style
+.+ 1
+.- 2
+* how to organize the compilations 
+* naming conventions -- how do we make a fun hyperlinked index 
+* what sort of ways do we want to look up recipes in the final product?
+* what sorts of files / builds should be more public -- ie how to share to others outside of potluck with a link for read? or
```
Comment (kunalsahasrabuddhe) [inline README.md]: Use github.io for things that face the world?

File: README.md (hunk 13 above)
Comment (dfudenb) [reply README.md]: yeah
-- was definitely a thought plus

File: README.md (hunk 14 above)
Comment (dfudenb) [reply README.md]: ah -- crossed out to taste and pinches, and pecs -- but the tex will do the full definition,

File: README.md (hunk 16 above)
Comment (dfudenb) [reply README.md]: then the question becomes how much minipage manipulation will people be doing -- 

the other question is will we ever want to allow more than one recipe on the same page -- and my inclination is towards no.

File: README.md [hunk 20]
```diff
@@ -12,19 +16,54 @@ for likely eventual transoformation into some physical entity.
...
+* how to reference other recipes from recipes
+* support for multiple page recipes, 
+* # of columns to use in the style
+.+ 1
+.- 2
+* how to organize the compilations 
+* naming conventions -- how do we make a fun hyperlinked index 
+* what sort of ways do we want to look up recipes in the final product?
+* what sorts of files / builds should be more public -- ie how to share to others outside of potluck with a link for read? or 
+* assume that this repository could be public at any time
```
Comment (kunalsahasrabuddhe) [inline README.md]: Assume that this repo is public at all times, unless we specifically want to keep it private (in which case we move to another platform and sacrifice certain features. In the dancebook project I mentioned we have dual review, which we gave up when they changed the free level features, but if we're doing single reviews only, that might be ok).

File: README.md (hunk 16 above)
Comment (dfudenb) [reply README.md]: at least definitely when in pdf form

File: README.md [hunk 21]
```diff
@@ -12,19 +16,54 @@ for likely eventual transoformation into some physical entity.
...
+* standards for how we cite inspiration
```
Comment (kunalsahasrabuddhe) [inline README.md]: Cite in the flavour text that describes why a dish was picked for a theme, or such? This'll need a bit of expansion, but I like keeping this in.

File: README.md (hunk 17 above)
Comment (dfudenb) [reply README.md]: cause lots of recipe formats put ingredients one side then the steps in the other, 
versus always ingredients above the steps 
then there are the story and tags and pictures which can also move around with the ingredients,

File: images/readme.md [hunk 22]
```diff
@@ -0,0 +1,3 @@
+# directory for images 
+
+splitting out will allow sparse checkouts if we need to light-weight the images -- however may be better to instead just keep them next to the recipes.
```
Comment (kunalsahasrabuddhe) [inline images/readme.md]: Splitting out also makes things easier with compilation of recipes across subdirs.

File: README.md (hunk 18 above)
Comment (dfudenb) [reply README.md]: yes -- however the top tex files can share a massive common amount of header.

File: README.md (hunk 19 above)
Comment (dfudenb) [reply README.md]: ?
details

File: README.md [hunk 23]
```diff
@@ -20,21 +20,21 @@ to be included in the final binding, as such we will discuss some (probably mult
...
-.* estimated servings?
-.* estimated time (total, elapsed, active)
-.* commentary and [flavor text]("not neccessarily about the actual flavor")
-.* required equipment (desired, necessary)
-.* date first brought to potluck (or `BGC` if before git-cookin) and the theme of that day if applicable
-.* source of inspiration / credit?
+..* title
+..* author
+...* this shouldn't need an example...
+..* tags
```
Comment (kunalsahasrabuddhe) [inline README.md]: The more I think about it, the more I think we should standardise tags early. We have enough history with these recipes that this won't be crazy.

File: README.md (hunk 20 above)
Comment (dfudenb) [reply README.md]: i mean we could eventually move to more review but--- biggest thing is content first.

File: README.md (hunk 8 above)
Comment (dfudenb) [reply README.md]: should only be consistent -- across all files the author authors --> which should be some \cmd that can be changed out then an authors -- naming convention file?

File: README.md (hunk 15 above)
Comment (dfudenb) [reply README.md]: yep 👍

File: README.md (hunk 21 above)
Comment (dfudenb) [reply README.md]: ah here i meant cite by like saw something in these two cookbooks or links and now synthesis, 

but inspiration is a good point as well, --

File: images/readme.md (hunk 22 above)
Comment (dfudenb) [reply images/readme.md]: i'm less pro this directory now --> and would instead have multiple sub image folders in the recipes/author/images
form?

File: README.md (hunk 23 above)
Comment (dfudenb) [reply README.md]: .... tags are always crazy. -- can only come up with a blessed list -- that you are strongly recommended to use if applicable

File: recipes/readme.md (hunk 3 above)
Comment (kunalsahasrabuddhe) [reply recipes/readme.md]: Yeah I think I'm on board with separation by chef. Benefits are obvious, overhead is low, as you say:
- Easy to compile by chef if desired, then, either at top level or using makefiles in every dir.
- Doesn't change overall file and repo architecture.

File: collations/readme.md (hunk 5 above)
Comment (kunalsahasrabuddhe) [reply collations/readme.md]: I think the default action after compiling (at least at time of review) should be to clean and remove all pdf files, except in the collations folder. There we keep blessed versions if we have something public-facing (even though the repo is public-facing too). That way source directories don't get cluttered but we know the compile works before review.

Maybe I'll also pull the SOP file from the other book project. It's a beast but no way for anyone to go wrong there if they follow it!

Am I going off-topic here?

File: README.md (hunk 6 above)
Comment (kunalsahasrabuddhe) [reply README.md]: Mm, ok. If we went with a database we could have this auto-populate from info fed into a database? db would also make it easier to curate sub-sampled collections in pdf form.

File: collations/readme.md (hunk 5 above)
Comment (dfudenb) [reply collations/readme.md]: sure sounds good -- 

thing seems reasonable -- however -- 
collations may not be output
but rather the input --> so another folder?
``editions?``

File: README.md (hunk 8 above)
Comment (kunalsahasrabuddhe) [reply README.md]: Database would make this easier too. Makes entering things a bit more complicated but allows the stylefile to dictate how things compile.

Thoughts on how complicated entering things into a db would be? Do we want to have the db pull from tex files to keep things consistent? My experience with database management is basically zero, let alone incorporating into tex makefiles, so I'm not sure I'm the right person to decide this architecture.

File: README.md (hunk 9 above)
Comment (kunalsahasrabuddhe) [reply README.md]: ??

File: README.md (hunk 10 above)
Comment (kunalsahasrabuddhe) [reply README.md]: For the former I think all images in a single images folder. Compile will sort all that out.

Maybe we make the stylefile on a couple of test recipes, then make sure that future changes to stylefiles don't break backward compatibility? I figure updates to stylefile are needed only if a corner case can't be worked around or ends up happening enough times that we need a better way to deal with it.

File: README.md (hunk 11 above)
Comment (kunalsahasrabuddhe) [reply README.md]: Impressions are useful to have. If a recipe makes it in here it's obviously been well reviewed, so this is more important if it was a redo that produced an amazing outcome.

This is secondary to the basic database/csv question, IMO - once we have that it should be easy enough to record this info.

File: README.md (hunk 6 above)
Comment (dfudenb) [reply README.md]: possibly -- not quite sure how everything would integrate -- 
but yes a db helps network everything

and not sure what sort of hosting would be entailed

File: README.md (hunk 16 above)
Comment (kunalsahasrabuddhe) [reply README.md]: Agree with no more than one recipe on a page. Shouldn't happen in non-pdf since every recipe will be its own source file anyway.

Hopefully a good stylefile means people won't even have to look at minipages :smile:

File: README.md (hunk 17 above)
Comment (kunalsahasrabuddhe) [reply README.md]: Yes true.

I guess we'll just develop our own format and build that into the stylefile?

File: README.md (hunk 8 above)
Comment (dfudenb) [reply README.md]: yerh -- not entirely sure -- 

then again this route isn't clear how complicated either -- 

and would entail a different set of access control and hosting or

mm

File: README.md (hunk 9 above)
Comment (dfudenb) [reply README.md]: ❗️

File: README.md (hunk 10 above)
Comment (dfudenb) [reply README.md]: 👍  images folder -- approved extensions must be called out

File: README.md (hunk 11 above)
Comment (dfudenb) [reply README.md]: derp

File: README.md (hunk 17 above)
Comment (dfudenb) [reply README.md]: yeah -- but want to think on that a bit now 
-- perhaps do some hand doodles --

File: README.md (hunk 16 above)
Comment (dfudenb) [reply README.md]: true story --

File: recipes/readme.md (hunk 3 above)
Comment (dfudenb) [reply recipes/readme.md]: :+1:

File: README.md (hunk 18 above)
Comment (kunalsahasrabuddhe) [reply README.md]: Hmm I think this actually goes up to what the makefile includes and what's left in the tex files. This is not something I'm clear on because my other book project doesn't compile all docs into a single pdf, but rather generates separate pdfs (I think), but if we want to generate a book with links within it, then either we compile using pdflatex and \include{} and ditch the idea of a makefile, or develop make-fu and latex-fu to do this through a makefile

File: README.md (hunk 19 above)
Comment (kunalsahasrabuddhe) [reply README.md]: I guess this is called github pages now. This allows you to use content on your github repo to make a webpage/website.

File: README.md (hunk 20 above)
Comment (kunalsahasrabuddhe) [reply README.md]: Agreed.

File: images/readme.md (hunk 22 above)
Comment (kunalsahasrabuddhe) [reply images/readme.md]: Reason?

File: README.md (hunk 18 above)
Comment (dfudenb) [reply README.md]: yeh -- so -- lets think about how we organize a cookbook

i mean the index and table of contents are both quite useful -- and knowing if things get broken down into sauces or other features
where in the book by reference and hyperref is quite nice
so that suggests to me that the whole jobbie has to get compiled at the same time. for editions 

and the final glossary is also interesting for perhaps having entries that have tag lookup -- eg these are all the dairy recipes, on pages clickme clickme clickme

Comment (dfudenb) [reply README.md]: what are the bennies of a makefile here ?

File: README.md (hunk 19 above)
Comment (dfudenb) [reply README.md]: ah yes

Comment (dfudenb) [reply README.md]: so we could also compile individual recipes somehow for git pages, meh

Comment (dfudenb) [reply README.md]: there will be links eventually -- 
but no hurry now
-- just suggests that there may be some place to put individual recipe output pdfs.

File: collations/readme.md (hunk 5 above)
Comment (kunalsahasrabuddhe) [reply collations/readme.md]: Yep agreed.

File: README.md (hunk 6 above)
Comment (kunalsahasrabuddhe) [reply README.md]: Yeah this is complicated.

File: README.md (hunk 11 above)
Comment (kunalsahasrabuddhe) [reply README.md]: I feel like we're saying the same thing in different ways?

I guess my point was, if recipe is bad, it won't be here, and if impressions are good we'll nominate it for the cookbook, so maybe impressions not relevant? Maybe only relevant if it's polarising?

File: README.md (hunk 10 above)
Comment (kunalsahasrabuddhe) [reply README.md]: Ah yes. Thoughts on approved extensions? png, jpg, tiff, eps, pdf?

File: README.md (hunk 17 above)
Comment (kunalsahasrabuddhe) [reply README.md]: I like the idea of hand doodles :smile:

File: README.md (hunk 21 above)
Comment (kunalsahasrabuddhe) [reply README.md]: Ok let's keep this point as a catch-all then

File: README.md (hunk 10 above)
Comment (dfudenb) [reply README.md]: in that folders readme

File: README.md (hunk 23 above)
Comment (kunalsahasrabuddhe) [reply README.md]: I like your initial list of tags here. Since we have "allergen" as a tag, maybe the stylefile will have an allergen field so we can list them?

File: README.md (hunk 6 above)
Comment (dfudenb) [reply README.md]: or if possibly several json / yaml files might be more sustainable than a csv if we are doing it locally in the db -- though probably someone has done something like that before just -- meh

File: README.md (hunk 11 above)
Comment (dfudenb) [reply README.md]: ah gotcha

Comment (dfudenb) [reply README.md]: so the flow will be more like someone requests a recipe of a dish
then the chef fills in a recipe file.
so pre blessed

Comment (dfudenb) [reply README.md]: more or less

File: README.md (hunk 6 above)
Comment (kunalsahasrabuddhe) [reply README.md]: Yeah I thought about json. Not familiar with yaml. This will require more reading on how best to implement.

File: README.md (hunk 8 above)
Comment (kunalsahasrabuddhe) [reply README.md]: Again maybe json instead?

File: README.md (hunk 9 above)
Comment (kunalsahasrabuddhe) [reply README.md]: .

File: README.md (hunk 11 above)
Comment (kunalsahasrabuddhe) [reply README.md]: Yeah basically. But if chef chooses to put it in without a request maybe it won't pass review?

Comment (kunalsahasrabuddhe) [reply README.md]: (And if so, is that a helpful policy?)

File: README.md (hunk 6 above)
Comment (dfudenb) [reply README.md]: yaml is a superset of json -- 
but can be formatted prettier for human readability 
and supports # comments

File: README.md (hunk 8 above)
Comment (dfudenb) [reply README.md]: possibly
then everything is right in git -

File: README.md (hunk 9 above)
Comment (dfudenb) [reply README.md]: !?!?

File: README.md (hunk 11 above)
Comment (dfudenb) [reply README.md]: well right now everything requires a PR to master 
so everything must get some minimal review

File: README.md (hunk 18 above)
Comment (kunalsahasrabuddhe) [reply README.md]: Yeah of course the whole thing gets compiled all at once. My point is I know how to do this with includes, but that doesn't necessarily extend to an index.

I like the idea of tag lookup in the glossary.

bennies - benefits? Mostly I think to do with filtering, since they're just bash scripts. But maybe latex can do that within .tex files, in which case the benefit is minimal

File: README.md (hunk 11 above)
Comment (dfudenb) [reply README.md]: also keeps the main timeline clear of crud - dy comments

File: README.md (hunk 17 above)
Comment (dfudenb) [reply README.md]: let me find some hand doodling implements

File: README.md (hunk 6 above)
Comment (kunalsahasrabuddhe) [reply README.md]: Ah yes. Could do yaml too, then :stuck_out_tongue:

File: README.md (hunk 23 above)
Comment (dfudenb) [reply README.md]: yeh -- though -- allergens are just preformatted tags -- for some ''alergens'' aren't really a thing for some people -- like meat as an allergen... 
but could be helpful to put it as its own section to help people understand why to fill out.

File: README.md (hunk 8 above)
Comment (kunalsahasrabuddhe) [reply README.md]: aye. And no databases to deal with even if within git(hub)

File: README.md (hunk 9 above)
Comment (kunalsahasrabuddhe) [reply README.md]: Ok now I'm so confused.

File: README.md (hunk 11 above)
Comment (kunalsahasrabuddhe) [reply README.md]: yeah of course, but I mean,ifit's not requested it doesn't pass review even if it otherwise meets the requirements?

File: README.md (hunk 18 above)
Comment (dfudenb) [reply README.md]: gotcha
i guess i considered the tex files as explicit listings of include file include files ish

Comment (kunalsahasrabuddhe) [reply README.md]: Topic for a future commit.
